packages/frontend/web/components/Outbrain.tsx

```diff
@@ -0,0 +1,29 @@
+// tslint:disable:react-no-dangerous-html
+
+import React from 'react';
+import { Container } from '@guardian/guui';
+
+export const shouldDisplayOutbrain = (config: ConfigType): boolean => {
+    return config.switches.outbrainDCRTest;
+};
+
+export const OutbrainWidget: React.FC<{}> = ({}) => {
+    return (
+        <div className="js-outbrain">
+            <div className="js-outbrain-container" />
+        </div>
+    );
+};
+
+export const OutbrainContainer: React.FC<{
+    config: ConfigType;
+}> = ({ config }) => {
+    if (shouldDisplayOutbrain(config)) {
+        return (
+            <Container>
+                <OutbrainWidget />
+            </Container>
+        );
+    }
+    return <div />;
```

Comment:
An alternative way to do this is the early-return style. I.e.

    if (!shouldDisplayOutbrain) {
        return null;
    }

    ..

This is very unidiomatic in Scala, but has some advantages in less pure languages, the main one being that the main path of code doesn't have to be indented/wrapped in an if.

Reply:
Done :)